packages/marketplace/src/components/ui/developer-analytics/cost-explorer/cost-explorer-component/cost-explorer-table.tsx

```diff
@@ -5,11 +5,19 @@ import { useSelector } from 'react-redux'
 import { selectMonthlyBilling, selectMonthlyBillingLoading } from '@/selector/developer'
 import { BillingBreakdownForMonthV2Model, ServiceItemBillingV2Model } from '@reapit/foundations-ts-definitions'
 
-export const prepareTableData = (data: ServiceItemBillingV2Model[]) => {
+export const prepareTableData = (data: ServiceItemBillingV2Model[], serviceName?: string) => {
   if (!data || !data.length) return []
 
-  return data.map(({ items = [], ...row }) => {
-    return { ...row, subRows: prepareTableData(items) }
+  return data.map(({ items = [], itemCount, ...row }) => {
```

Comment:
I saw no changes in test file. Recomment to write a test which service is `API Requests` to cover this case